src/mz/src/main.rs

```diff
@@ -195,18 +182,22 @@ const PROFILE_NOT_FOUND_MESSAGE: &str =
 #[tokio::main]
 async fn main() {
     let args = Cli::parse();
-    let profile_arg: Option<String> = args.profile;
+    let mut profile_name = DEFAULT_PROFILE_NAME.to_string();
+    let preferedined_profile_arg: Option<String> = args.profile;
+    let predefined_profile_env: Option<&str> = option_env!("MZ_PROFILE");
+
+    if let Some(profile_env) = predefined_profile_env {
```

Comment:
Clap supports both defaults and env vars, so this should be removed and the appropriate clap options configured.